lms/djangoapps/commerce/api/v1/views.py

```diff
@@ -3,16 +3,13 @@
 from django.contrib.auth.models import User
 from django.http import Http404
 from edx_rest_api_client import exceptions
-from edx_rest_framework_extensions.authentication import JwtAuthentication
-from rest_framework.authentication import SessionAuthentication
 from rest_framework.generics import ListAPIView, RetrieveUpdateAPIView
-from rest_framework.permissions import IsAuthenticated
 from rest_framework.views import APIView
-from rest_framework_oauth.authentication import OAuth2Authentication
```

Comment:
It's great to remove these auth/permissions classes from all over the place and centralize them in the decorator code.